component/src/main/java/com/celements/blog/plugin/NewsletterReceivers.java

```diff
@@ -19,6 +19,10 @@
  */
 package com.celements.blog.plugin;
 
+import static java.nio.charset.StandardCharsets.*;
```

Comment:
@ebeutler Because without static we cannot access the method .name() used in the method nr. 442 encode(String, UTF-8.name()) to convert the Charset to String because the encode() doesn't accept Charset.